src/pages/partners/payment-agent/_your-control.tsx

```diff
@@ -19,42 +19,45 @@ const Card = styled(Flex)`
     width: auto;
     justify-content: flex-start;
     align-items: center;
-
     @media ${device.laptop} {
         height: 120px;
     }
-
     @media ${device.tabletL} {
-        flex-direction: column;
-        padding: 24px;
+        display: flex;
+        flex-direction: row;
+        padding: 24px 24px 32px 24px;
+        align-items: center;
         width: 328px;
-        min-height: 184px;
-        height: auto;
+        min-height: 96px;
+        height: 120px;
     }
     @media ${device.mobileM} {
-        width: 100%;
+        height: 100%;
```

Comment:
Will these heights cater for all languages?

Reply:
Yes. Manually checked it